packages/code-du-travail-frontend/src/common/Article.js

```diff
@@ -82,3 +100,15 @@ const Content = styled.div`
     margin-top: ${spacings.base};
   }
 `;
+
+const HideOnMobile = styled.span`
```

Comment:
Did we already have this kind of component somewhere ? it could belong to the react-ui

Reply:
Yes je l'ai copié collé lui mais je ne sais pas si je suis prêt à faire un composant juste pour remplacer une media query. Je me tate encore sur le besoin

Reply:
la règle des 3 copié / collé avant de refactor... on garde ;)